src/main/python/hydra/lib/hydrabase.py

```diff
@@ -418,6 +418,19 @@ def create_app_instances_group(self, app_name, group_name, num_app_instances, an
         app_grp.add_tasks_to_group(tasks_list)
         return app_grp
 
+    def delete_app_instances_group(self, group_name):
+        """
+        Delete a particular tasks group.
+        @args:
+        group_name:          Name of the app instances' group
+        :return:             return 0 in case of 0. 1 otherwise.
+        """
+        if group_name not in self.app_group:
+            l.error("Group with this name does NOT exist")
+            return 1
```

Comment:
Usually, pythonic is to return True and False, yes they get translated to 1 and 0 but still :)